plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -640,4 +641,14 @@ private class DokkaDescriptorVisitor(
         Visibilities.PRIVATE -> KotlinVisibility.Private
         else -> KotlinVisibility.Public
     }
+
+    private fun fallbackPackageName(): String = "[${sourceSet.sourceSetName} root]"// TODO: error-prone, find a better way to do it
+}
+
+fun DRI.withPackageFallbackTo(fallbackPackage: String): DRI {
```

Comment:
Move it to DRI.kt or make it private if you want to treat it as a local utility, to avoid scope pollution.